travelwise/frontend/views.py

```diff
@@ -107,8 +107,9 @@ def create_plan(request):
     activities = []
     for stored_activity in Activity.objects.filter(plan=travel_plan).order_by('start_time'):
         activities.append({
-            'start_time': stored_activity.start_time,
-            'end_time': stored_activity.end_time,
+            'id': stored_activity.pk,
```

Comment:
Note: id and pk are assigned the same in django. This is fine.

Reply:
I see, I didn't realize. I've used pk everywhere lol. We should clean that up in a separate PR if its that important, since its used more than right here.